control-plane/config/crd/bases/consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -75,13 +75,22 @@ spec:
                     minimum: 1
                     type: integer
                 type: object
+              mapPrivilegedContainerPorts:
+                description: The value to add to privileged ports ( ports < 1024)
+                  for gateway containers
+                format: int32
+                type: integer
```

Comment:
uh-oh, did the generators not get re-run? these fields seems to be from the addition of some openshift specific stuff, which isn't in-scope in this PR.

Reply:
Potentially. I just ran them for my CRD changes.

Reply:
Looking at what changed, I think the previous compilation used the wrong version of KubeBuilder. The version we should be using is `0.8.0`. I think the previous change was made with `0.11`

Reply:
can we only include the CRD changes that are necessary for this PR and clean up the rest as part of a PR against `main`?

Reply:
Yes, I will manually fix these.